doc/mgr/dashboard.rst

```diff
@@ -681,7 +682,7 @@ ways:
 
 #. Use both sources simultaneously.
 
-All three methods are going to notify you about alerts. You won't be notified
+All three methods notify you about alerts. You won't be notified
```

Comment:
Personally, I'd change "won't" to "will not"

Reply:
agreed.